CONTRIBUTING.md

```diff
@@ -145,23 +145,11 @@ Scrub build results:
     make lint
     make dco
 
-### Restore, update and save dependencies
+### Managing dependencies
 
-When you make a fresh copy of the repo, all the dependecies are in `vendor/` directory for the builds to work. If you want to update the dependencies
-
-#### Restore the dependencies
-
-    make dep-restore
-
-   This uses godep to restores all the dependencies to your `$GOPATH`. Note that this changes the packages in your `$GOPATH`
-
-#### Add one ore more dependencies
-
-    go get -u <new dependency>
-
-#### Save the dependencies to `vendor/`
-
-    make dep-save
+When you make a fresh copy of the repo, all the dependecies are in `vendor/` directory for the builds to work.
```

Comment:
s/builds/build/

Reply:
btw there are multiple options to build I guess. That's why there was 'builds'.